Cargo.toml

```diff
@@ -71,3 +71,15 @@ path = "crates/fj-viewer"
 [workspace.dependencies.fj-window]
 version = "0.47.0"
 path = "crates/fj-window"
+
+[profile.dev.package.'*']
+opt-level = 3
+
+[profile.dev]
+opt-level = 1
+lto = false
+
+[profile.release]
+opt-level = 3
+lto = true
+strip = true
```

Comment:
Why did you choose those specific settings? They look very deliberate, but I can't tell why they might be better than others. I'm specifically wary of the `dev` settings, as those have the potential to make iteration times during development worse.